src/axom/mint/docs/sphinx/index.rst

```diff
@@ -3,14 +3,88 @@
+
+* Support for 1D/2D/3D mesh geometry.
+
+* Efficient data structures to represent :ref:`ParticleMesh`,
+  :ref:`StructuredMesh` and :ref:`UnstructuredMesh` types, including
+  unstructured meshes with :ref:`MixedCellTopology`.
+
+* Native support for a variety of commonly employed :ref:`CellTypes`.
+
+* A flexible :ref:`MeshStorageManagement` system, which can optionally
+  inter-operate with `Sidre <../../../sidre/docs/sphinx/index.html>`_ as the underlying, in-memory, hierarchical
+  datastore, facilitating the integration across packages.
+
+* Basic support for :ref:`sections/fem`, consisting of
+  commonly employed *shape functions* and *quadratures*.
+
+* A Mesh-Aware :ref:`sections/execution_model`, based on the `RAJA <https://raja.readthedocs.io/en/main/index.html>`_ programming
+  model abstraction layer that supports on-node parallelism for mesh-traversals,
+  enabling the implementation of computational kernels that are born parallel
+  and portable across different processor architectures.
+
+
+Requirements
+------------
+
+Mint is designed to be *light-weight* and *self-contained*.
+The only requirement for using Mint is a C++11 compliant compiler.
+However, to realize the full spectrum of capabilities, support for
+the following third-party libraries is provided:
+
+* `RAJA <https://raja.readthedocs.io/en/main/index.html>`_, used for the parallel execution and portability layer.
+
+* `Conduit <https://llnl-conduit.readthedocs.io/en/latest/index.html>`_ , for using `Sidre <../../../sidre/docs/sphinx/index.html>`_ as the :ref:`MeshStorageManagement` system.
+
+* `Umpire <https://umpire.readthedocs.io/en/develop/>`_, for memory management on next-generation architectures.
+
+For further information on how to build the `Axom Toolkit <../../../../index.html>`_ using these
+third-party libraries, consult the `Axom Quick Start Guide <../../../../docs/sphinx/quickstart_guide/index.html>`_.
+
+
+About this Guide
+----------------
+
+This guide discusses the basic concepts and architecture of Mint.
+
+* The :ref:`sections/mint/getting_started` section provides a quick introduction
+  to Mint, designed to illustrate high-level concepts and key capabilities, in
+  the context of a small working example.
+
+* The :ref:`sections/tutorial` section provides code snippets that
+  demonstrate specific topics in a structured and simple format.
+
+* For complete documentation of the interfaces of the various classes and
+  functions in Mint consult the `Mint Doxygen API Documentation <../../../../doxygen/html/minttop.html>`_.
+
+* Complete examples and code walk-throughs of mini-apps using Mint are
+  provided in the :ref:`sections/examples` section.
+
+Additional questions, feature requests or bug reports on Mint can be submitted
+by `creating a new issue on Github <https://github.com/LLNL/axom/issues>`_
+or by sending e-mail to the Axom Developers mailing list at axom-dev@llnl.gov.
 
 .. toctree::
-   :caption: Table Of Contents
+   :titlesonly:
```

Comment:
Just by comparison to the index page of other components
 

- Is a `:caption: Contents` missing here as in other components?
- Should the `:hidden:` be removed (I don't remember why I added that)

The navigation bar on the left only expands one level. Any ideas?

Reply:
the expansion is due to the top level toctree (i think).  If it was hidden i removed the caption because it didnt show.  I am okay with going across all components and unhiding them and adding :caption: Contents so its more uniform.

Reply:
I am in favor of consistency. I thought that the `:maxdepth:` attribute in this file was specifying the levels of expansion.

If not, could we change that in the top level toctree?